src/index.js

```diff
@@ -12,13 +13,35 @@ function addEventsDragAndDrop(el) {
   el.addEventListener('dragend', dd.dragEnd, false);
 }
 
+function editUI(show, index) {
+  if (show) {
+    const formDiv = document.getElementById('editForm');
+    formDiv.classList.add('w-100');
+    formDiv.classList.add('h-100');
+    formDiv.classList.add('text-center');
+    formDiv.innerHTML = "<form action='' class='form  text-center w-100 h-100 m-auto'><br><div class='title-border-top'></div><h2 class='add-book-title text-white'>Edit description</h2><br><input type='text' name='title' id='title' placeholder='Title'><br><br><div class='form-btn'> <button type='submit' class='text-center ' id='add-button'>Change</button></div></form>";
+
+    const addButton = document.getElementById('add-button');
+    addButton.addEventListener('click', () => {
+      const title = document.getElementById('title').value;
+      addremove.edit(title, index);
+      editUI(false);
+    });
+  } else {
+    const formDiv = document.getElementById('editForm');
+    formDiv.innerHTML = '';
+  }
+}
```

Comment:
[OPTIONAL]

- I would recommend to reset the form after adding a toDO list. Kindly the form stays the same.

![Screenshot from 2021-07-15 18-54-25](https://user-images.githubusercontent.com/22434660/125795473-e1688b49-51e4-4c9a-8b9a-60151465f095.png)